packages/auth/src/passwordEncryption.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import { randomBytes } from 'crypto';
+import { hash, verify } from '@node-rs/argon2';
+import sha256 from 'sha256';
+
+/**
+ * Helper function to encrypt passwords using argon2
+ * @param password {string}
+ * @param salt {string}
+ * @returns {Promise<string>}
+ */
+export function encryptPassword(password, salt) {
+  return hash(`${password}${salt}`);
```

Comment:
This isn't necessary, Argon2 already includes generating a salt

https://github.com/napi-rs/node-rs/blob/main/packages/argon2/src/lib.rs#L126